src/social-network/utils/getCurrentNetworkWorker.ts

```diff
@@ -1,15 +1,8 @@
 import { definedSocialNetworkWorkers, SocialNetworkWorker } from '../worker'
 import { GroupIdentifier, Identifier, PersonIdentifier, PostIdentifier, PostIVIdentifier } from '../../database/type'
-import { env } from '../shared'
 
-function find(network: string) {
-    return (v: SocialNetworkWorker) => {
-        // noinspection SuspiciousTypeOfGuard
-        if (typeof v.networkIdentifier === 'string') return v.networkIdentifier === network
-        // @ts-ignore   this is a robust design since sometimes non-string id may supported.
-        return v.networkIdentifier(network, env, {})
-    }
-}
+export const find = (network: string) => (v: SocialNetworkWorker) => v.networkIdentifier === network
```

Comment:
export this function with an over-simplified name?

Reply:
`isOnNetwork`

Reply:
Or not export it at all

Reply:
That's an error. No need to export.

Reply:
fixed 6477765aecb0a11bb6cfcbb9c2c3886ca4dd03ec